lib/screens/lobby/lobby.dart

```diff
@@ -1,12 +1,16 @@
 import 'package:flutter/material.dart';
-import 'package:niira/models/game.dart';
-import 'package:niira/navigation/navigation.dart';
-import 'package:niira/screens/lobby/created_game_tile.dart';
+import 'package:geolocator/geolocator.dart';
```

Comment:
I reckon you're be better off removing the geolocator dependency - it's a clean architecture thing, happy to have a crack at explaining the benefits if you want, not sure if I'll do a good job or not. I can point you to a good reference if that's more helpful. Uncle Bob's books would be the best source I'd say, it's mainly the 'D' in the SOLID principles that's relevant here I'd say, off the top of my head... "Depend on abstractions, not on concretions."  [SOLID Principles made easy | Hacker Noon](https://hackernoon.com/solid-principles-made-easy-67b1246bcdf)

Reply:
So you'd have to make the LocationService convert (the geolocator) Position to (your homemade) Location objects so your return types can use Location (or something else that is your own PODO)

Reply:
But it's your call of course!

Reply:
thanks for pointing it out! Super happy to implement that 😄